hack/.ci/set_dependency_version

```diff
@@ -108,7 +108,7 @@ elif name == 'vpn':
 elif name == 'external-dns-management':
     names = ['dns-controller-manager']
 elif name == 'logging':
-    names = ['fluentd-es', 'curator-es']
+    names = ['loki']
```

Comment:
yes, sure

Reply:
Yes, @ialidzhikov is correct. This script is used to find and update the correct image-entry in `charts/images.yaml`. For that it assumes that the image's `name` is equal to the component to be upgraded. Where this does not hold (or several images are part of one component) this part of the script adjusts the mapping on the fly.
From a quick glance at the changed `images.yaml`, the following should be correct:
```suggestion
    names = ['fluent-bit']
```
Loki does not need to be added since it isn't built by logging.